### PR TITLE
Update Dispatch EP codes

### PR DESCRIPTION
**Pull request by joofsh**
This PR updates EP codes to match the VBMS update. The old codes are no longer used. Currently, usage of Caseflow Dispatch has stopped until this change gets merged.

connects  #2012 

Testing:
- [x] Updated all tests

### CODE REVIEW

File: spec/feature/establish_claim_spec.rb [hunk 1]
```diff
@@ -459,45 +459,6 @@
 
         expect(task.reload.completion_status).to eq("special_issue_emailed")
       end
-
-      context "When there is an existing 172 EP" do
```
Comment (joofsh) [inline spec/feature/establish_claim_spec.rb]: We are no longer limited to 2 full grant EP, so removing this test

File: client/app/components/TextField.jsx [hunk 2]
```diff
@@ -94,8 +94,8 @@ TextField.propTypes = {
   required: PropTypes.bool.isRequired,
   type: PropTypes.string,
   validationError: PropTypes.string,
-  value: PropTypes.oneOfType(
+  value: PropTypes.oneOfType([
```
Comment (joofsh) [inline client/app/components/TextField.jsx]: unrelated fix, was causing console errors

File: spec/feature/establish_claim_spec.rb [hunk 3]
```diff
@@ -476,45 +476,6 @@
...
-              benefit_claim_id: "1",
-              claim_receive_date: 10.days.ago.to_formatted_s(:short_date),
-              claim_type_code: "172GRANT",
-              end_product_type_code: "172",
-              status_type_code: "PEND"
-            }
-          ]
-        end
-
-        scenario "Assigning it to complete the claims establishment" do
```
Comment (shanear) [inline spec/feature/establish_claim_spec.rb]: dont we still need a test that assigns an existing EP to a task?
